sdk/monitor/OpenTelemetry.Exporter.AzureMonitor/README.md

```diff
@@ -0,0 +1,34 @@
+# Azure Monitor Exporter client library for .NET
```

Comment:
To pass CI, we need name in a format Azure. + client library for .NET. Will research later on how to avoid this check.